lib/msf/core/handler/reverse.rb

```diff
@@ -55,6 +55,17 @@ def bind_port
         (port > 0) ? port : datastore['LPORT'].to_i
       end
 
+      # Checking if LHOST is a loopback address
+      def is_loopback_address(address)
```

Comment:
And for the record, this code is @busterb's: https://github.com/rapid7/metasploit-framework/pull/8063#discussion_r104604972.